src/Numerics/Mesh/Grids.jl

```diff
@@ -655,6 +657,50 @@ function computegeometry_fvm(elemtocoord, D, ξ, ω, meshwarp)
             num_vgeo_handled += 1
         end
 
+        # Compute Δx3, the radial grid size for finite volume method, 
+        # if vertical finite volume is applied in the last dimension
+        if Nq[dim] == 1
```

Comment:
I don't think that this is quite right if only one of the horizontal dimensions is `N = 0`. I know we do not currently plan to allow this in the kernels, but the mesh stuff generally supports any dimension being `N = 0`, so I'd like to future proof this. (even if its with filling with `NaN` when it will break.

Reply:
This is only used for vertical direction N=0 case

Reply:
<img width="477" alt="Screen Shot 2020-12-02 at 8 52 01 AM" src="https://user-images.githubusercontent.com/16932196/100904435-e145c580-347b-11eb-9ba6-423bca6d4827.png">
<img width="477" alt="Screen Shot 2020-12-02 at 8 52 28 AM" src="https://user-images.githubusercontent.com/16932196/100904445-e30f8900-347b-11eb-8d0a-6e36566ea9c3.png">

Reply:
I mean that the code doesn't work if polynomial order is something like `(0, 3, 0)`

Reply:
Yes, the code works only for (n,n,0) I think we need to require first two are the same ?

Reply:
My point is the mesh generator supports any numbers, so I'd like this to do something reasonable in all cases. (Even if it's filling with `NaN` in cases we do not currently handle.

Reply:
Just update a version to handle (0,3,0)